### ISSUE TITLE
V+N+content clause idioms: "have no idea", "give a damn", "make up one's mind"

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#444]
The remaining obj+ccomp instances from #303 are V+N idioms with a content clause: http://universal.grew.fr/?custom=63ec582b711b0

- I'm just trying to make up my mind **whether I should move to Australia**
- I don't give a damn **what adults do to one another**
- He obviously had no idea **what he was talking about**

Is there a better solution than obj + ccomp for these?

**Comment by amir-zeldes** [comment]
Maybe dislocated (two realizations of same slot), or obl:npmod for the NP? The last one could also be adnominal on idea somehow (appos, nmod:npmod or similar)

**Comment by nschneid** [comment]
"My mind was not made up whether to move to Australia" works, no? And the "whether" clause is optional, so it's hard to square that with obl:npmod.

I don't think the definition of `dislocated` fits well either ("fronted or postposed elements that do not fulfill the usual core grammatical relations of a sentence").

**Comment by amir-zeldes** [comment]
Yeah, I don't think it's great either, I was just assuming obj+ccomp is strictly forbidden. I guess iobj is another way to go, but it seems to me obj+ccomp would have been better (if it's allowed)

**Comment by nschneid** [comment]
"have no idea/clue": see also #300
